docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+"""
+Train Test Validation Scenario on Lending Club Data - 5 min Quickstart
+***********************************************************************
+
+The deepchecks train test validation suite is relevant any time you have data splits
```

Comment:
I don't like the `data splits` wording, maybe we can find a better term?

Reply:
Maybe do use the terms "reference" and "evaluation" or something like that here. Leakage & drift check are there to help you understand if using the second dataset for evaluation is OK.

Reply:
I think reference and evaluation is limiting (to the specific scenario when it's train and test)